packages/shorebird_cli/lib/src/shorebird_env.dart

```diff
@@ -169,4 +169,8 @@ class ShorebirdEnv {
       return null;
     }
   }
+
+  /// Whether the CONTINUOUS_INTEGRATION environment variable is set to true.
+  bool get isRunningOnCI =>
```

Comment:
`flutter` tool has some fancier logic here.
https://github.com/search?q=repo%3Aflutter%2Fflutter%20CONTINUOUS_INTEGRATION&type=code

Reply:
What a pile of 💩 
https://stackoverflow.com/questions/9738535/powershell-test-for-noninteractive-mode
https://stackoverflow.com/questions/6108330/checking-for-interactive-shell-in-a-python-script

Reply:
I saw this (and this is where I got the approach and `CONTINUOUS_INTEGRATION` flag name from), but wasn't sure how much of it we wanted to follow. Should I just plop the flutter file in third_party, or is checking for all the same keys in our source acceptable?

Reply:
Checking the same keys is acceptable.

Reply:
Updated!